search/reverse_geocoder.cpp

```diff
@@ -276,7 +276,7 @@ void ReverseGeocoder::GetNearbyBuildings(m2::PointD const & center, double radiu
   {
     std::string const & hn = GetHouseNumber(ft);
     if (hn.empty())
-      return;
+      std::string const & hn = "";
```

Comment:
What was the intention here?

Reply:
I thought that hn could be null, and I was trying to set it to an empty string. But actually, hn cannot be null. I will simply remove the if--return.